MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -899,23 +898,24 @@ private void BuildStreamVideoItem(
             string? audioCodec)
         {
             // Prefer matching video codecs
-            var videoCodecs = ContainerProfile.SplitValue(videoCodec);
-
-            // Enforce HLS video codec restrictions
-            if (playlistItem.SubProtocol == MediaStreamProtocol.hls)
+            List<string> videoCodecs = [];
+            if (videoCodec is not null)
+            {
+                var splitStrings = videoCodec.Split(',');
+                videoCodecs.AddRange(splitStrings);
+            }
+            else if (videoStream is not null)
```

Comment:
Hmm, what if `videoCodec` is not null, but gives an empty array?
```suggestion
            }

            if (videoCodecs.Length == 0 && videoStream is not null)
```